.env

```diff
@@ -0,0 +1 @@
+REACT_APP_INSTANT_ENABLED=1
```

Comment:
do we need to commit changes like this?

Reply:
This was cherry-picked from your commit. We can revert it, but might complicate merging.

Reply:
Ah, yes nvm. I thought it's just changing line endings.